### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Namrata-J**
#2 here in this pr i have added auth pages UI using mui v5, and have further added functionality to the same using mockbee.

Please review this PR, thankyou!!

### CODE REVIEW

**Review by Amandeep-sinha — APPROVED** [review]
Looks good to merge 👍

File: src/components/authPageComponents/AuthCardInputField.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    return (
+        <TextField
+            error={errorAttr(inputProps.uniqueId, inputProps.text)}
+            onChange={(e) => handleChange(e, inputProps.uniqueId)}
+            sx={authCardInput}
+            helperText={inputProps.text}
+            id={inputProps.uniqueId}
+            label={inputProps.inputLabel}
+            type={inputProps.inputType}
```
Comment (sushil-mod) [inline src/components/authPageComponents/AuthCardInputField.jsx:17]: If possible you can destructure input props, then you don't have to write inputProps. every time

**Review by sushil-mod — APPROVED** [review]
Auth pages look Nice. Good Code .just a suggestion

File: src/components/authPageComponents/AuthCardInputField.jsx (hunk 1 above)
Comment (Namrata-J) [reply src/components/authPageComponents/AuthCardInputField.jsx:17]: yeah, thanks for pointing that out, will work on your suggestion!!
thankyou so much for reviewing!!

**Pull request merged**